src/cljs/velho_ds/molecules/field.cljs

```diff
@@ -299,39 +307,47 @@
                                  (swap! state assoc :selected-from-filter ""))
                                (when (= key "Tab")
                                  (swap! state assoc :focus false)))
+        click-in-dropdown? (fn click-in-dropdown? [element id]
+                             (if (nil? element)
```

Comment:
Could this be done with (when (not (nil? element) ? If fn returns nil it equals to false?